samples/bluetooth/cap_initiator/README.rst

```diff
@@ -0,0 +1,74 @@
+.. zephyr:code-sample:: bluetooth_cap_initiator
+   :name: Bluetooth: Common Audio Profile Initiator
+   :relevant-api: bt_cap bt_bap bluetooth
+
+   Bluetooth: Common Audio Profile Initiator
```

Comment:
```suggestion
   Send audio data to a Bluetooth device using the Common Audio Profile (CAP) Initiator role
```

Reply:
@Thalley  was there a reason not to use the suggested wording? For the sake of consistency with other samples I deliberately proposed a wording aligned with the recommendations in https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/templates/sample.tmpl

Reply:
It doesn't send audio data 8) It's just dummy data that cannot be played back